011-kafka-canary.md

```diff
@@ -0,0 +1,40 @@
+# Strmixi Canary
+
+Implement a Kafka canary tool which will act as an indicator of whether Kafka clusters are operating correctly. This would be achieved by creating a canary topic and continously producing and consuming events on the topics. 
+
+## Current situation
+
+Currently  Strimzi does not have any canary style feature. It does have readiness checks for Kafka cluster which can be replaced/augmented with the canary.
+
+## Motivation
+
+The canary would provide the following:
+  
+  * Improved readiness check for Kafka clusters
+  * SLI(s) for service admins
+  * Metrics:
+    * End-to-end latency
+    * Succesfull/failed events
+    * Disk usage
```

Comment:
I may be lacking strimzi or kafka knowledge,
but how do you see the canary providing disk metrics?
That seems like more of a host level metric that the product/consumer or even the broker

Reply:
Yeah in the current dashboard we get this kind of information from the kubelet volume related metrics.

Reply:
Based this on a comment from @tombentley in another channel but I may have misunderstood. Is this possible @tombentley?